### DISCUSSION TITLE
[Sharing] Ability to be found by apps as a "Gallery"

### DISCUSSION THREAD

**Discussion by herculespt** [immich-app/immich#14344] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've been using Immich since the beginning. I love your work, and I use it to myself and my wife.
I found that would be amazing if we can use it as a kind of gallery that can be found by other apps.

Example: In WhatsApp we can't choose Immich to select or send a picture.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1641
